### PR TITLE
update test articles for easier testing and fix randomizer to include…

### PR DESCRIPTION
**Pull request by daleinen7**
… all articles

[ticket](https://trello.com/c/0LEX6Q9T/124-random-case-studies-is-not-working-correctly)

### CODE REVIEW

File: _posts/2019-03-09-Proxy-post-3.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Lorem ipsum dolor sit amet, consectetur
```
Comment (MangeFre) [inline _posts/2019-03-09-Proxy-post-3.md:2]: Any reason we need to merge in the testing post titles into master? I think it'd look better to keep the Lorem Ipsum etc as is.

File: _layouts/post.html [hunk 2]
```diff
@@ -15,7 +15,7 @@ <h1>{{ page.title }}</h1>
         <img src="{{page.img}}" alt="{{page.alt}}">
         {% else %} <img src="/images/proxy-img.png" alt="A sample proxy image">
         {% endif %}
-        <p class="image-caption">{{ page.imgtext}}</p>
+        <p class="image-caption">{{ page.imgtext }}</p>
       </div>
```
Comment (MangeFre) [inline _layouts/post.html:19]: This did the trick, works well now!

**Review by MangeFre — CHANGES_REQUESTED** [review]
Looks all good, just thinking it would look cleaner to keep the posts on master the same with the "lorem ipsums" across the board. Maybe it does not matter, but you would mind changing that back to how it was?

File: _posts/2019-03-09-Proxy-post-3.md (hunk 1 above)
Comment (daleinen7) [reply _posts/2019-03-09-Proxy-post-3.md]: Makes sense. No problem.

**Review by MangeFre — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**